leakcanary-android-core/src/main/java/leakcanary/internal/tv/TvToast.kt

```diff
@@ -28,15 +29,18 @@ internal object TvToast {
     activity: Activity,
     text: CharSequence
   ): Toast {
-    val toast: Toast = Toast.makeText(activity, text, Toast.LENGTH_LONG)
+    val inflater = LayoutInflater.from(activity)
+    val toast = Toast(activity)
 
-    // Custom toast views are deprecated from API level 30 on, see
-    // https://developer.android.com/reference/android/widget/Toast#getView()
-    if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.Q) {
-      val textView = toast.view!!.findViewById<TextView>(android.R.id.message)
-      textView.setCompoundDrawablesWithIntrinsicBounds(R.drawable.leak_canary_icon, 0, 0, 0)
-      textView.compoundDrawablePadding =
-        activity.resources.getDimensionPixelSize(R.dimen.leak_canary_toast_icon_tv_padding)
+    val iconSize = activity.resources.getDimensionPixelSize(
+      R.dimen.leak_canary_toast_icon_size
+    )
+    toast.apply {
+      setGravity(Gravity.CENTER_VERTICAL, 0, -iconSize)
+      duration = Toast.LENGTH_LONG
+      val inflatedView = inflater.inflate(R.layout.leak_canary_heap_dump_toast, null)
+      inflatedView.findViewById<TextView>(R.id.leak_canary_toast_text).text = text
```

Comment:
Use `viewBinding` instead of `viewBinding` here?

Reply:
The project does not use `viewBinding`

Reply:
@pyricau May should intro `viewBinding` in?

Reply:
No need for core library. For sample project It just doesn't make sense. anyways I guess it's out of the scope of this PR and could be discussed in the separate ticket with Py

Reply:
Let's not add `viewBinding` into the library, if there is no need for it.